test/common/upstream/outlier_detection_impl_test.cc

```diff
@@ -133,15 +135,15 @@ TEST_F(OutlierDetectorImplTest, DestroyWithActive) {
       cluster_, empty_outlier_detection_, dispatcher_, runtime_, time_source_, event_logger_));
   detector->addChangedStateCb([&](HostSharedPtr host) -> void { checker_.check(host); });
 
-  loadRq(cluster_.hosts_[0], 4, 503);
+  loadRq(cluster_.hosts_[0], 4, 500);
```

Comment:
Don't we want to preserve the old behavior for 5xx (including 503) when `enforcingConsecutiveGatewayFailure()` is not enabled? Otherwise, we're making a breaking change here, which is subject to the deprecation policy https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md.

Reply:
Yes, and we do, however using a 503 in these tests will cause a logEject call (with enforced = false) for the ConsecutiveGatewayFailure as well. I felt that switching to 500 preserved what this test was trying to test, and left the test cleaner than if I'd added EXPECT_CALL for that additional call.